src/Public.Api/Infrastructure/Startup.cs

```diff
@@ -175,11 +175,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
 
                             options.InvalidModelStateResponseFactory = actionContext =>
                             {
-                                var format =
-                                    actionContext.GetValueFromHeader("format")
-                                    ?? actionContext.GetValueFromRouteData("format")
-                                    ?? actionContext.GetValueFromQueryString("format");
-
+                                var format = actionContext.DetermineFormatParameter();
                                 var acceptType = actionContext.HttpContext.Request.GetTypedHeaders().DetermineAcceptType(format, actionContext.ActionDescriptor);
```

Comment:
I think you can pull this out of `format.ContentType` and if you add AcceptType to `ContentFormat` too (you already have it I think) that as well.